apps/web/lib/features/task/task-status.tsx

```diff
@@ -657,15 +669,18 @@ export function TaskStatus({
 	return (
 		<div
 			className={clsxm(
-				'py-2 md:px-4 px-2 flex items-center text-sm space-x-0 rounded-xl relative',
+				'py-2 md:px-3 px-2 flex items-center text-sm font-[500] text-dark space-x-3 relative',
```

Comment:
@desperado1802 Put rounded-xl dynamically, you can pass it from the parent component to use it in both the homepage and sidebar.